api/handler/acl.go

```diff
@@ -158,17 +159,31 @@ func (h *handler) gateKey(ctx context.Context) (*keys.PublicKey, error) {
 	return gateKey, nil
 }
 
+func (h *handler) bearerTokenIssuerKey(ctx context.Context) (*keys.PublicKey, error) {
```

Comment:
I suppose we can use hex encoded key value rather than `*keys.PublicKey`. I just don't like explicit `elliptic.P256()` mention.
Though we use not only hex encoded key value but also address of it. But it's used as display name and only for one grantee (there is not consistency). So we can omit it.

Any thoughts, @alexvanin

Reply:
> I just don't like explicit elliptic.P256() mention.

Accessbox, neofs-node and neo-go widely use this explicit curve declaration, so we are tightly connected with NIST P256 anyway. At least we have nice `key.Address()`